interfaces/prompting/constraints_test.go

```diff
@@ -465,7 +465,7 @@ func (s *constraintsSuite) TestAbstractPermissionsFromAppArmorPermissionsUnhappy
 		},
 		{
 			"home",
-			notify.AA_MAY_GETATTR | notify.AA_MAY_READ,
```

Comment:
This was testing that getting an unexpected permission (previously `AA_MAY_GETATTR`) would cause the correct error. And `AA_MAY_GETCRED` is still not mapped, so the idea was to replace `AA_MAY_GETATTR` with `AA_MAY_GETCRED`. So not exactly being dropped, just trying to come up with permissions for which the `"home"` interface has no mapping.